website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -288,26 +288,37 @@ Expert Mode Keyboard Shortcuts:
 
 **Most users should start by [running the standalone OpenBCI GUI](#running-the-openbci_gui). The Processing IDE is for adding features, modifying existing code, or testing development versions.**
 
-The OpenBCI GUI was built using [Processing](https://processing.org/), a popular, Java-based creative coding framework. If you are interested in adding features or modifying the existing code, it is really easy to do so if you are familiar with Java. If you're not familiar with Java, don't fret! Processing is one of the easiest software packages to pick up as a beginner coder.
+The OpenBCI GUI is built using [Processing 4](https://processing.org/), a popular, Java-based creative coding framework. If you are interested in adding features or modifying the existing code, it is really easy to do so if you are familiar with Java. If you're not familiar with Java, don't fret! Processing is one of the easiest software packages to pick up as a beginner coder.
 
 The things you will need to run the OpenBCI GUI in Processing are:
 
-- [Processing App](https://processing.org/download/?processing)
+- [Processing App](https://processing.org/download)
 - [OpenBCI GUI Sketch](https://github.com/OpenBCI/OpenBCI_GUI)
 
-First, go to processing.org and download the latest version of Processing. [Here is a walkthrough on how to install Processing on Ubuntu Linux.](http://www.artsnova.com/processing/installing-processing-ubuntu-linux-tutorial.html)
+### Installing Processing 4
 
-Go ahead and move it to your `Applications` folder, and launch the application. If this is the first time that you are running Processing, it will create what it calls it's `Sketch` folder in the following locations:
+First, go to Processing's website and [download the latest version of Processing](https://processing.org/download). [Here is a walkthrough on how to install Processing on Ubuntu Linux.](http://www.artsnova.com/processing/installing-processing-ubuntu-linux-tutorial.html)
 
-On a Mac `Users/<user-name>/Documents/Processing`  
- On Windows `C:\Users\<user-name>\Documents\Processing`  
- On Linux `/home/<user-name>/sketchbook`
+:::noteMac Users
```

Comment:
This usage of custom admonition may change when updating Docusaurus.

Reply:
to

:::note{Custom Message]

Reply:
Decided to remove this and restructure this section for such reasons. It's cool, but it would be the first time this is done and I don't feel like digging to fix this later.